section3.py

```diff
@@ -0,0 +1,29 @@
+from typing import Callable, Any
+
+import decorator
+
+
+@decorator.decorator
+def change_functionality(func: Callable,  *args: Any, **kwargs: Any) -> int:
+    """
```

Comment:
The name of this function is not indicative. 
Also notice you're not performing the functionality the assignment specifies.